manual/en/docs/flow/actions/regex/regex_replace_action.md

```diff
@@ -0,0 +1,28 @@
+# Regex string replace plugin
+
+This plugin takes string as an input and replaces its fragments that match
+given regex.
+
+## Input
+This plugin takes any payload as input.
+
+## Output
+This plugin returns regex after operations on port **replaced** if matches were found,
+or given string on port **not_found** if given value was not containing any fragment
+matching given regex.
+
+## Configuration
+
+#### With form
```

Comment:
This should be in Form. Make the description simple in the form. Now It is too wordy. Example: Do not type `Here Provide ...`, only `Type a path to the string you want to substite`.